crates/codegen/parser_runtime/src/support/stream.rs

```diff
@@ -1,6 +1,6 @@
 use std::ops::Range;
 
-use super::text_index::TextIndex;
+use super::super::text_index::TextIndex;
```

Comment:
nit: For core/general enough structs I feel that using `crate` imports is cleaner than having to use super (more than once)

Reply:
I prefer the relative paths, but it's definitely just taste.

Reply:
unrelated to this PR: I also prefer using `crate::*` because:

1. It is easier for the IDE to suggest imports without breaking the file by inserting `use` statements in the wrong place, or removing unused imports from a single statement automatically.
2. It is easy to understand what other parts of the crate are being used in a module if they are all grouped in the same statement.
3. Perhaps most important, the files survive moving around in the crate, without having to rewrite `use` statements. This is critical for codegen and using `#[path]`, since `crate::x::y::z` is valid everywhere in the crate.

But ultimately I want to configure `rustfmt` and Rust Analyzer (at some point in the future) to automatically apply/enforce a single standard across the codebase, without us having to manually review/modify them. I think we currently use both styles all over the place.

Reply:
ok, not an important preference for me, and I see your points.